src/utils/utils.ts

```diff
@@ -316,7 +317,10 @@ export function getFunctionTypeString(node: FunctionDefinition, compilerVersion:
   const inputs = node.vParameters.vParameters
     .map((decl) => {
       const baseType = getNodeType(decl, compilerVersion);
-      if (baseType instanceof ArrayType || baseType instanceof UserDefinedType) {
+      if (
+        baseType instanceof ArrayType ||
+        (baseType instanceof UserDefinedType && !(baseType.definition instanceof EnumDefinition))
```

Comment:
This should be  a positive check for StructDefinition not a negative check for EnumDefinition, since Contract types can have default storage location